File: StockMarketWatcherBackend/Stock.cs
namespace StockMarketWatcherBackend
{
    public class Stock
    {
        public string Name { get; set; }
        public float PriceNow { get; set; }
        public float StartingPrice { get; set; }
        public List <float> Diff { get; set; }

        public Stock(string name, float startingPrice, float priceNow, List<float> diff)
        {
            Name = name;
            PriceNow = priceNow;
            StartingPrice = startingPrice;
            Diff = diff;
        }
    }
}

File: app/model/Stock.js
Ext.define('MarketMaven.model.Stock', {
    extend: 'MarketMaven.model.Base',

    store: 'MarketMaven.store.StockStore', // Provides data from backend
    fields: [
        {name: 'name', type: 'string'},
        {name: 'priceNow', type: 'float'},
        {name: 'startingPrice', type: 'float'},
        {name: 'diff', type: 'float'}
    ]
});